Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp

```diff
@@ -1074,13 +1074,18 @@ namespace ScriptCanvasEditor
             ? *AZStd::any_cast<AZ::EntityId>(connectionUserData)
             : AZ::EntityId();
 
-        if (AZ::EntityUtils::FindFirstDerivedComponent<ScriptCanvas::Connection>(scConnectionId))
+        if (ScriptCanvas::Connection* connectionComponent = AZ::EntityUtils::FindFirstDerivedComponent<ScriptCanvas::Connection>(scConnectionId))
```

Comment:
Nit: Can specify type as "auto" for readability here since the type is already being passed to the templated method